internal/provisioner/kops_utils.go

```diff
@@ -50,10 +55,14 @@ func updateKopsInstanceGroupAMIs(kops *kops.Cmd, kopsMetadata *model.KopsMetadat
 	var ami string
 	for _, ig := range instanceGroups {
 		if ig.Spec.Image != kopsMetadata.ChangeRequest.AMI {
+			if val, ok := ig.Metadata.Labels[SkipAMIUpdateLabelKey]; ok {
```

Comment:
Should you check `val` for truthy here?

Reply:
We could, but the value really doesn't matter so I decided to keep it simple for now.